doc/user/Container-Image.md

```diff
@@ -20,9 +20,16 @@ curl -L https://download.geofabrik.de/europe/germany/berlin-latest.osm.pbf -o be
 # download GTFS
 curl -L https://vbb.de/vbbgtfs -o berlin/vbb-gtfs.zip
 # build graph and save it onto the host system via the volume
-docker run --rm -v "$(pwd)/berlin:/var/opentripplanner" docker.io/opentripplanner/opentripplanner:latest --build --save 
+docker run --rm \
+    --memory=8g \
+    -e JAVA_OPTS='-Xmx8g' \
```

Comment:
I know it's common but a few paragraphs down we say that we want to use JAVA_TOOL_OPTIONS. I might remove support for JAVA_OPTS in the future.